### ISSUE TITLE
Unknown prop `allowNewLine` on <textarea>

### ISSUE THREAD

**Issue by mafzst** [nikgraf/belle#321]
Hi!

Nice work!

A little problem (non blocking), React throw warning when I'm using TextInput component:
```
warning.js:44 Warning: Unknown prop `allowNewLine` on <textarea> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in textarea (created by TextInput)
    in TextInput (created by Form)
    in form (created by Form)
    in div (created by Card)
    in Card (created by Form)
    in Form (created by Connect(Form))
    in Connect(Form) (created by Header)
    in header (created by Header)
    in Header (created by Connect(Header))
    in Connect(Header) (created by MainApp)
    in div (created by MainApp)
    in MainApp
    in div (created by AppRoot)
    in Provider (created by AppRoot)
    in div (created by AppRoot)
    in AppRoot
```

**Comment by HugoHeneault** [comment]
Any ideas on this? :)

**Comment by mcbryant** [comment]
I gave this and other similar issues [a shot](https://github.com/nikgraf/belle/pull/323).

**Comment by nikgraf** [comment]
@mafzst @HugoHeneault just published 2.0.8 with @mcbryant's fixes
